flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/parse/CalciteParser.java

```diff
@@ -57,6 +58,15 @@ public SqlNode parse(String sql) {
         }
     }
 
+    public SqlNodeList parseSqlList(String sql) {
```

Comment:
Add some javadoc and also add a `throws` to specify you throw the flink's exception, rather than the calcite one